docs/install_guides/installing-openvino-apt.md

```diff
@@ -145,7 +145,7 @@ Step 2: Install OpenVINO Runtime Using the APT Package Manager
    
          .. code-block:: sh
    
-            sudo apt install openvino-2023.1.0
+            sudo apt install openvino-2022.3.1
```

Comment:
The goal is to provide the latest LTS version for user to work with, @kblaszczak-intel can you provide more info on it?

Reply:
why only apt / rpm are exception here?

I propose to not overcomplicate the things and keep latest state.

Reply:
We have two cases here. One is installing the latest package and one is installing a specific package. It's safe to assume the selected version is different than the most recent one. Otherwise, the two options would result in installing the same version. Such instruction may be confusing to the reader. That's why we want to differentiate between them, indicating that a past version may be selected, and it should be an LTS release, so as not to advertise prior versions from the current cycle.

Reply:
> It's safe to assume the selected version is different than the most recent one

But why do we need to recommend to install not the latest one? 
Modern features are available in latest release, so we need to demonstrate latest one IMO

Reply:
We're not recommending it, we're making it available. If we don't want anyone to install versions other than the latest one, should consider removing the "install specific package" option from the article? If not, we'd like to differentiate between the two, so that we don't confuse the reader with two instructions for the same result.

Reply:
we want to highlight that users can install different versions, but it's not mandatory to have different version as an example.
Moreover, it will not work to install 2022.x from 2023 apt repository:
```sh
echo "deb https://apt.repos.intel.com/openvino/2023 ubuntu22 main"
```
You can see that link contains 2023 in its name => it' provides only 2023.x packages.

Reply:
I don't see any problem in demonstration of installation of current version. 
Out users are smart enough to understand they can replace it with older one